book/src/SUMMARY.md

```diff
@@ -1,21 +1,36 @@
 # Kubernetes/CSI Book
 
-- [Introduction](Home.md)
-- [Using a CSI Driver](UsingCSIDriver.md)
+- [Introduction](introduction.md)
+- [Developing CSI Driver for Kubernetes](developing.md)
+    - [Sidecar Containers](sidecar-containers.md)
+        - [external-provisioner](external-provisioner.md)
+        - [external-attacher](external-attacher.md)
+        - [node-driver-registrar](node-driver-registrar.md)
+        - [cluster-driver-registrar](cluster-driver-registrar.md)
+        - [external-snapshotter](external-snapshotter.md)
+        - [livenessprobe](livenessprobe.md)
+    - [CSI CRDs](csi-crds.md)
+      - [CSIDriver Object](csi-driver-object.md)
+      - [CSINodeInfo Object](csi-noide-info-object.md)
+    - [Features](features.md)
+        - [Secrets & Credentials](ephemeral-local-volumes.md)
+        - [Volume Snapshot & Restore](Implement-Snapshot-Feature.md)
+        - [Topology](topology.md)
+        - [Raw Block Volume](block.md)
+        - [Skip Attach](skip-attach.md)
+        - [Pod Info on Mount](pod-info.md)
+        - [Ephemeral Local Volumes](ephemeral-local-volumes.md)
+- [Deploying CSI Driver on Kubernetes](deploying.md)
+- [Driver Testing](testing-drivers.md)
+    - [Driver - Unit Testing](unit-testing.md)
+    - [Driver - Functional Testing](functional-testing.md)
+- [Drivers](Drivers.md)
```

Comment:
some of the pages start with capital letters, others lowercase.  It would be good to be consistent

Reply:
Only the sidecar container names start with lower case to make it easy to recognize them by repo names, everything else is upper case.

Reply:
sorry i was referring to the page names, like Drivers.md vs introduction.md